granulate_utils/linux/cgroups/cgroup.py

```diff
@@ -28,32 +34,43 @@
 }
 
 
-def get_cgroups(process: Optional[psutil.Process] = None) -> List[Tuple[str, List[str], str]]:
+@dataclass
+class ProcCgroup:
+    hier_id: str
+    cgroup_relative_path: str
+    controllers: List[str]
```

Comment:
`cgroup_relative_path` is incorrect terminology. The path _is_ the cgroup. Also, the order should be id, controllers, path. Refer to `cgroup(7)`:

>The colon-separated fields are, from left to right:
>
>1. For cgroups version 1 hierarchies, this field contains a unique hierarchy ID number that can be matched to a hierarchy ID in /proc/cgroups.  For the cgroups version 2 hierarchy, this field contains the value 0.
>
>2. For cgroups version 1 hierarchies, this field contains a comma-separated list of the controllers bound to the hierarchy.  For the cgroups version 2 hierarchy, this field is empty.
>
>3. This field contains the pathname of the control group in the hierarchy to which the process belongs.  This pathname is relative to the mount point of the hierarchy.